src/cloud-api-adaptor/docker/Makefile

```diff
@@ -0,0 +1,53 @@
+AA_KBC  ?= offline_fs_kbc
```

Comment:
Sounds like duplicate with the Makefile in podvm/podvm-mkosi, maybe we can move the file docker/Dockerfile to podvm/Dockerfile.podvm.docker and reuse the file podvm-mkosi/Makefile. Sounds like need refine podvm-mkosi/Makefile also. @bpradipt it's not must but up to you.

Reply:
The only difference is that for docker provider I am using Ubuntu binaries due to the dependency on Kind image (based on ubuntu) for now. I didn't want to extend the podvm-mkosi Makefile since we anyway wanted to standardise on a single distro. Unfortunately for docker provider this is not possible without working on a Fedora based image to run nested containers. I plan to work on it once this initial support is merged. Then I can remove these docker provider specific Makefile and rely on podvm-mkosi/Makefile itself. 
Is it ok ?